### PR TITLE
added documentation of early returns

### PR DESCRIPTION
**Pull request by sjka**
Signed-off-by: Simon Kaufmann <simon.kfm@googlemail.com>

### CODE REVIEW

File: configuration/rules-dsl.md [hunk 1]
```diff
@@ -207,6 +207,19 @@ Besides the implicitly available variables for items and commands/states, rules
 - Every rule that has at least one command event trigger, will have the variable `receivedCommand` available, which can be used inside the execution block.
 - Every rule that has at least one status change event trigger, will have the variable `previousState` available, which can be used inside the execution block.
 
+### Early returns
+
+Similar to Java, it is possible to return early from a rule, not executing the rest of the statements like this:
```
Comment (ThomDietrich) [inline configuration/rules-dsl.md]: Remove "Similar to Java" ?

File: configuration/rules-dsl.md [hunk 2]
```diff
@@ -207,6 +207,19 @@ Besides the implicitly available variables for items and commands/states, rules
...
+
+```java
+if (Temperature.state > 20) {
+	return;
+}
+sendCommand(Heating, ON)
+```
+
+Caveat: Please note the semicolon after the return statement. If you leave it out, the next line might still get executed.
```
Comment (ThomDietrich) [inline configuration/rules-dsl.md]: "might" doesn't create confidence in a determined openHAB behavior. In my opinion just leave it with the first sentence or add something as a short rationale like "which terminates the command without an additional argument." wdyt?

**Comment by sjka** [comment]
Thanks for your feedback! I have updated the PR.

**Comment by ThomDietrich** [comment]
Thanks!

**Pull request merged**